docs/UserGuide.md

```diff
@@ -640,14 +665,15 @@ Use this command to edit the specified event in your NeoBook.
 
 #### Important requirements
 
-`INDEX` must be a **positive integer** (must be greater than 0).
+1)`INDEX` must be a **positive integer** (must be greater than 0).
+2) At least one field must be provided.
```

Comment:
`1), 2), ...` do not render properly on the site or PDF, might wanna change to `1., 2., ...`